dist/locales/en.json

```diff
@@ -10708,10 +10708,6 @@
                 "name": "OpenStreetMap Galicia Twitter",
                 "description": "Follow us on Twitter at {url}"
             },
-            "Galicia-Xeoinquedos-Twitter": {
```

Comment:
@quincylvania I ended up introducing a lot of diffs related to the community index and NSI after rebasing and building; might want to double-check that I’m not blowing anything away due to something missing in my configuration.

Reply:
Hmm you probably need to run `npm install` to sync up your versions of these dependencies before building again.

Reply:
Thanks, that worked.